docs/rules/no-skip-test.md

```diff
@@ -0,0 +1,27 @@
+# Ensure no tests are skipped
+
+It's easy to make a test skipped with test.skip() and then forget about it. It's visible in the results, but still easily missed.
```

Comment:
Maybe mention that this can potentially be disastrous if you deploy code and your whole test suite is pretty much not validating anything.